### DISCUSSION TITLE
Seems to be working perfectly on MSI Bravo 15!

### DISCUSSION THREAD

**Discussion by Inurian** [YoCodingMonster/OpenFreezeCenter#123] [General]
I have only tested cooler-boost so far, because I'm generally using intensive enough things on this laptop that I need it at almost all times, but if I can help confirm some more things, let me know!

**Comment by Ritul13** [comment]
Can you help me i am using arch linux distro garuda and it is not running at all i am doing everything in it my vant get any help
